en-US/Characters_Codex/06_Spellcasting/Spell_Descriptions/S/Shapechange.md

```diff
@@ -1,5 +1,22 @@
 #### Shapechange
-<!-- TODO Check and tag this spell-->
+<!-- markdownlint-disable link-image-reference-definitions -->
+[_metadata_:spell_name]:- "Shapechange"
+[_metadata_:spell_level]:- "9"
+[_metadata_:spell_school]:- "transmutation"
+[_metadata_:ritual]:- "false"
+[_metadata_:casting_time_amount]:- "1"
+[_metadata_:casting_time_unit]:- "action"
+[_metadata_:range]:- "Self"
+[_metadata_:target]:- "Self"
+[_metadata_:components_verbal]:- "true"
+[_metadata_:components_somatic]:- "true"
+[_metadata_:components_material]:- "true"
+[_metadata_:components_material_description]:- "a jade circlet worth at least 1,500 gp, which you must place on your head before you cast the spell"
+[_metadata_:components_material_cost]:- "1,500 gp"
+[_metadata_:duration]:- "1 hour"
+[_metadata_:concentration]:- "true"
+[_metadata_:compared_to_wotc_srd_5.1]:- "mechanics_different_wording_different"
```

Comment:
If we change the [gear stuff](https://github.com/wyrmworkspublishing/free5e/pull/264/files#r2326129444), it would be `mechanics_same_wording_different` here.